src/main/java/mate/academy/spring/service/mapper/CinemaHallDtoMapperService.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.CinemaHallResponseDto;
+import mate.academy.spring.model.CinemaHall;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallDtoMapperService {
```

Comment:
```suggestion
public class CinemaHallDtoMapper {
```
let's remove `Service` from the name of all mappers